Evaluation.Rmd

```diff
@@ -245,16 +245,17 @@ While `source3()` is considerably more concise than `source2()`, this one use ca
 1.   We can make `base::local()` slightly easier to understand by spreading
     out over multiple lines:
 
-    ```{r}
-    local3 <- function(expr, envir = new.env()) {
-      call <- substitute(eval(quote(expr), envir))
-      eval(call, envir = parent.frame())
-    }
-    ```
+
+```{r}
```

Comment:
These indents are nessary for correct alignment of the code with the rest of the exercise